src/actions/composite.js

```diff
@@ -25,7 +25,13 @@ export const updateApp = (change, router) => dispatch => {
   dispatch(updateFilters(change))
 
   if (router) {
-    history.push(createNewLocation({ change, router }))
+    const loc = createNewLocation({ change, router })
+    if (window.caches) {
```

Comment:
Good question, I think it would replace the old one with the new one from this function.

The point of this bit is so as a user changes the URL and goes across the site, those visited URLs are also cached to enable faster (and offline) access.